ooler_mqtt_bridge

```diff
@@ -137,6 +106,77 @@ async def control_power(mqtt: Client, myooler: ooler.Ooler) -> None:
             await send_update(mqtt, myooler)
 
 
+async def get_discovery_payloads(myooler: ooler.Ooler) -> Dict[str, Dict[str, str]]:
```

Comment:
Rather than this method here, probably it makes more sense to make this a function of the Ooler object itself. Thoughts?

Reply:
My thought was to have the Ooler class be generic and so shouldn't know about any of the MQTT bits